integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature

```diff
@@ -118,14 +118,11 @@ Feature: SmartMetering Configuration - Firmware
       | FirmwareModuleVersionFunc | M57 4836               |
     Then the update firmware result should be returned
       | DeviceIdentification      | TEST1024000000002      |
-      | FirmwareModuleVersionComm | Telit 10.00.154        |
-      | FirmwareModuleVersionMa   | BL_012 XMX_N42_GprsV09 |
-      | FirmwareModuleVersionFunc | M57 4836               |
-    And the database should be updated with the new device firmware
+    And the database should not be updated with the new device firmware
       | DeviceIdentification      | TEST1024000000002      |
-      | FirmwareModuleVersionComm | Telit 10.00.154        |
-      | FirmwareModuleVersionMa   | BL_012 XMX_N42_GprsV09 |
-      | FirmwareModuleVersionFunc | M57 4836               |
+      | FirmwareModuleVersionComm |                        |
+      | FirmwareModuleVersionMa   |                        |
+      | FirmwareModuleVersionFunc |                        |
```

Comment:
If the database should NOT be updated as the test step description suggests then the firmwaremoduleversion should have the same values as the values in the 'given' situation. And that is not empty, or is it?

Reply:
Yes, it was empty and it should stay empty